src/util/utils.js

```diff
@@ -58,4 +58,14 @@ function isArray(thing) {
     return (Object.prototype.toString.call(thing) === '[object Array]');
 }
 
-export {isFunction, isUndefined, isDefined, isObject, isArray, isString};
+/**
+ * Remove decimal place if the decimal value is empty.
+ *
+ * @param value
+ * @returns {*}
+ */
+function precision(value) {
+    return value % 1 === 0 ? 0 : null;
```

Comment:
`12312.123 % 1 = 0.12299999999959255`

We'll lose precision, `.123` will become `.122`.

Just send it currency formatter:
```js
currencyFormatter.format(1123123.1231, { code: 'USD' });

"$1,123,123.12"
``` 
Currency always has 2 decimal digits in most cases. I think we don't need to handle the precision while formatting.